frontend/src/actions/users.ts

```diff
@@ -64,13 +70,20 @@ export const fetchUserWithFollows = (token: string, id?: number | string) => {
   };
 };
 
-export const followUser = (userId: number, token: string) => {
-  return (dispatch: Dispatch) => {
+export const followUser = (data: {
+  user_id: number;
+  token: string;
+  callback: Function;
+}) => {
+  return async (dispatch: Dispatch) => {
     backend.get('/sanctum/csrf-cookie').then(async (csrf_response) => {
+      let user_id, token, callback;
+      ({ user_id, token, callback } = data);
```

Comment:
Can be done in one line
```suggestion
      let { user_id, token, callback } = data;
```

Reply:
done